utils/runtime/getting_failure_step.yml

```diff
@@ -0,0 +1,27 @@
+- block:
+
+    ## Getting failure step from experiment-pod
+    - name: Getting failure step from experiment pod
+      shell: >
+        kubectl logs {{ chaos_pod_name }} -n {{ a_ns }} | grep "FAILED!" -B 1 | head -1 | awk -F "Step:" '{print $2}'
+      register: fail_step
+      args: 
+        executable: /bin/bash
```

Comment:
Not compulsory, but can we use the same format(order) as we use for all the `shell` commands
```yaml
    ## Getting failure step from experiment-pod
    - name: Getting failure step from experiment pod
      shell: >
        kubectl logs {{ chaos_pod_name }} -n {{ a_ns }} | grep "FAILED!" -B 1 | head -1 | awk -F "Step:" '{print $2}'
      args: 
        executable: /bin/bash
      register: fail_step
```